src/components/header/style.css

```
@@ -16,4 +16,14 @@ header {
 
 h1 {
     font-size: 100px;
+}
+
+@media screen and (max-widtH: 800px) {
```

Caps on rule declaration
```suggestion
@media screen and (max-width: 800px) {
```